openprescribing/media/js/src/highcharts-options.js

```diff
@@ -179,6 +179,11 @@ barOptions.xAxis.type = 'category';
 barOptions.legend.enabled = false;
 barOptions.xAxis.labels = {
   formatter: function() {
+    if (typeof this.value === 'undefined') {
+      // We hope this will be logged by Sentry to give us some clues about why
+      // this.value is sometimes not defined.
+      console.log(barOptions);
```

Comment:
`console` isn't defined in IE unless the dev tools panel is open so this will throw another error, potentially confusing things.

Reply:
Oof, did not know that!